src/components/dashboard/main-content/course-enrollments/data/utils.js

```diff
@@ -132,6 +129,8 @@ export const getTransformedAllocatedAssignments = (assignments, slug) => {
       enrollBy: assignmentEnrollByDeadline,
       isCanceledAssignment,
       isExpiredAssignment,
+      assignmentConfiguration: item.assignmentConfiguration,
+      uuid: item.uuid,
```

Comment:
Docs nit: Would it help to document that these two additional keys are actually used by `useAcknowledgeContentAssignments`?  As it stands, the docstring suggests that the keys are intended for use by the CourseCard components only, but it seems they are multi-purpose now.

Reply:
Updated!